### PR TITLE
ITM-405:  Add quantized injuries

### PR DESCRIPTION
**Pull request by nextcen-dgemoets**
- Add quantized injuries
- Fixed edge case in evaluating conditions
- Emphasize most errors/warnings in logs

See the [ITM-405 client branch](https://github.com/NextCenturyCorporation/itm-evaluation-client/tree/ITM-405) and [PR](https://github.com/NextCenturyCorporation/itm-evaluation-client/pull/37).

To test, use the `itm_human_input.py` ADM simulator:
- `python itm_human_input.py --session test --scenario sample-1`

Verify the following in the client:
- `treatments_required` is always None
- Mike's pre-treated laceration shows a `treatments_applied` of `1` at the beginning of the scenario.
- Once you discover Mike's puncture injury (e.g, performing any action on him), each time you treat it successfully (right side, hemostatic gauze) the `treatments_applied` is incremented.
   - The first time you successfully treat it, the injury `status` changes from `discovered` to `partially treated`.
- Once you successfully treat it three times, the `status` changes to `treated`.

Feel free to try any other tests.

### CODE REVIEW

File: swagger_server/itm/itm_scenario_reader.py [hunk 1]
```diff
@@ -266,6 +266,9 @@ def _generate_character(self, character_data) -> Character:
                 location=injury.get('location'),
                 severity=injury.get('severity'),
                 source_character=injury.get('source_character'),
+                treatments_required=injury.get('treatments_required', 1),
+                #treatments_applied=injury.get('treatments_applied', 0), # Uncomment if/when sim allows partial pre-treatment
```
Comment (kaitlyn-sharo) [inline swagger_server/itm/itm_scenario_reader.py:270]: I'm sure we can do this if it's requested. Let me know if that happens

**Review by kaitlyn-sharo — APPROVED** [review]
Awesome

**Review by dereknop — COMMENTED** [review]
Everything looks good from the testing I did except for when I choose to do a sitrep on Mike the injury to his right side isn't revealing (doing a check vitals reveals the injury). Not sure if this is the intended behavior or a bug so I figured I'd mention it.

**Comment by nextcen-dgemoets** [comment]
> Everything looks good from the testing I did except for when I choose to do a sitrep on Mike the injury to his right side isn't revealing (doing a check vitals reveals the injury). Not sure if this is the intended behavior or a bug so I figured I'd mention it.

That's the intended behavior-- because Mike is unresponsive, he doesn't respond to SITREP so nothing new is learned about him (other than the fact that he is unresponsive).

**Review by dereknop — APPROVED** [review]

**Pull request merged**